packages/headless/src/features/commerce/pagination/pagination-actions.ts

```diff
@@ -4,15 +4,16 @@ import {validatePayload} from '../../../utils/validate-payload';
 
 const numberValue = new NumberValue({required: true, min: 0});
 
-export const selectPage = createAction(
-  'commerce/productListing/pagination/selectPage',
+export const setPageSize = createAction(
+  'commerce/pagination/setPageSize',
   (payload: number) => validatePayload(payload, numberValue)
 );
 
-export const nextPage = createAction(
-  'commerce/productListing/pagination/nextPage'
+export const selectPage = createAction(
+  'commerce/pagination/selectPage',
+  (payload: number) => validatePayload(payload, numberValue)
 );
 
-export const previousPage = createAction(
-  'commerce/productListing/pagination/previousPage'
-);
+export const nextPage = createAction('commerce/pagination/nextPage');
+
+export const previousPage = createAction('commerce/pagination/previousPage');
```

Comment:
This diff is a bit weird. We're just adding add new `setPageSize` action.

Reply:
Are we renaming from `productListing/pagination` to simply `pagination` because it will apply to all solution types?

Reply:
Yeah, it was a mistake on my part to have them contain `productListing` in the action name in the first place! The name of an action used across solution types should be agnostic of the solution type.